### ISSUE TITLE
[Feature Request] Modify List-Unsubscribe message header to forward through Addy

### ISSUE THREAD

**Issue by httpjamesm** [anonaddy/anonaddy#524]
Marketing emails have `List-Unsubscribe` headers that tell email clients where and what to send to unsubscribe from marketing lists. I'd like Addy to modify this so the recipient address for unsubscribing is a proxy address. This would allow unsubscribing using email clients without revealing your true address, as the unsubscribe message would go through Addy.

**Comment by obadz** [comment]
> This is a bad idea. Aliases can be for important accounts that deliver notifications too, not exclusively newsletters.

Absolutely. I'm not sure why that implies that "unsubscribe ⇒ block" is a bad idea though.
